packages/render/src/ui/index.tsx

```diff
@@ -130,6 +130,22 @@ function createDefaultComponents<TStylingProps extends Record<string, unknown>>(
     Image(props, stylingProps) {
       const aspectRatio = props.aspectRatio.replace(":", "/");
 
+      let sanitizedSrc =
+        props.status === "frame-loading" ? undefined : props.src;
+
+      // Don't allow data URLs that are not images -- we don't want to allow arbitrary data to be loaded
+      if (
+        sanitizedSrc?.startsWith("data:") &&
+        !sanitizedSrc?.startsWith("data:image")
+      ) {
+        sanitizedSrc = "";
+      }
+
+      // Don't allow SVG data URLs -- could contain malicious code
+      if (sanitizedSrc?.startsWith("data:image/svg")) {
+        sanitizedSrc = "";
```

Comment:
sanitizedSrc is not passed in as src to the `img` element

Reply:
fixed, ty